### ISSUE TITLE
sine.zig example fails to compile with "error: extern functions have no body"

### ISSUE THREAD

**Issue by shybyte** [ziglang/zig#5901]
When I try to compile the sine.zig example from https://ziglang.org/#Integration-with-C-libraries-without-FFIbindings using `zig build-exe sine.zig -lsoundio -lc`, I get the error message:

```
./sine.zig:27:1: error: extern functions have no body
extern fn write_callback(
```

**Comment by joachimschmidt557** [comment]
I think the `extern` should be replaced by `export`

**Comment by joachimschmidt557** [comment]
Yeah, upon further testing, the `sine.zig` example needs some updating to the latest zig syntax.

**Comment by joachimschmidt557** [comment]
Maybe this issue should be transferred to https://github.com/ziglang/www.ziglang.org
